src/components/MeetingsPage/MeetingsPage.jsx

```diff
@@ -0,0 +1,222 @@
+import { useEffect, useState, useContext } from 'react';
+import { usePortals } from 'react-portal-hook';
+import moment from 'moment';
+import { Context } from 'src';
+import { addMeeting, editMeeting, deleteMeeting } from 'src/services/MeetingService';
+import ConfirmEdit from 'src/components/ConfirmEdit/ConfirmEdit';
+import ConfirmDelete from 'src/components/ConfirmDelete/ConfirmDelete';
+import Header from 'src/components/Header/Header';
+import ErrorSnackbar from 'src/components/ErrorSnackbar/ErrorSnackbar';
+import deleteImg from 'src/img/delete.svg';
+import editImg from 'src/img/edit.svg';
+import './style.scss';
+
+const MeetingsPage = () => {
+  const store  = useContext(Context);
+  const portalManager = usePortals();
```

Comment:
nothing related to modal windows should be here, since this is a branch for displaying and adding appointments